qt/widgets/common/test/FitPropertyBrowserTest.h

```diff
@@ -53,4 +62,39 @@ class FitPropertyBrowserTest : public CxxTest::TestSuite {
     // Make sure the FunctionFactory does not have a dead link as an observer
     TS_ASSERT_THROWS_NOTHING(FunctionFactory::Instance().unsubscribe("FitPropertyBrowserTest_Funct");)
   }
+
+  void test_getOldExpressionAsString_returns_empty_string_when_tie_is_null() {
+    // initialise it fitPropertyBrowser- this adds an observer on the function factory update message
```

Comment:
I think you can remove the extra "it" from the comment here? And same on next test